pkgs/tools/security/rbw/default.nix

```diff
@@ -61,7 +58,7 @@ rustPlatform.buildRustPackage rec {
     patchShebangs bin/pass-import
     substituteInPlace bin/pass-import \
         --replace pass ${pass}/bin/pass
-  '' + lib.optionalString withGitCredential ''
+  '' + ''
     patchShebangs bin/git-credential-rbw
     substituteInPlace bin/git-credential-rbw \
         --replace rbw $out/bin/rbw
```

Comment:
Lets move that to the beginning before any conditional stuff.

Reply:
thanks.